tpl/template_embedded.go

```diff
@@ -213,7 +210,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
 {{ with .Site.Social.twitter }}<meta name="twitter:site" content="@{{ . }}"/>{{ end }}
 {{ with .Site.Social.twitter_domain }}<meta name="twitter:domain" content="{{ . }}"/>{{ end }}
 {{ range .Site.Authors }}
```

Comment:
I think you need to change the `.Site.Authors` to plain `.Authors` so that it refers to the `Authors` function in `page.go`. Similarly, I know there are other occurrences. You might want to grep for `.Site.Author` and check everyone of it closely once.